recipes/R/Retail - GradientBoosting/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # from Adobe.
 #####################################################################
 
-FROM adobe/acp-dsw-ml-runtime-r:0.15.0
+FROM adobe/acp-dsw-ml-runtime-r:0.17.0
 
 RUN install2.r gbm
 RUN install2.r lubridate
```

Comment:
Just a note here that we are trending towards pinning libs to versions that we have tested against to mitigate the 'shifting sand' effect of using unversioned dependencies.  Maybe on next update we can fix?
e.g.
```
RUN install2.r gbm==1.0.0
RUN install2.r lubridate==1.5.0
```